src/main/java/com/historycode/ui/page/adminpanel/jobspage/JobsPageGridComponent.java

```diff
@@ -11,9 +10,36 @@
 @Getter
 public class JobsPageGridComponent extends BaseGridComponent {
     List<JobsRowComponent> jobsRowComponents;
-    
+
     public JobsPageGridComponent(WebDriver driver, WebElement rootElement) {
         super(driver, rootElement);
```

Comment:
for(WebElement element: rowNodes ){
jobsRowComponents.add(new JobsRowComponent(driver, element))
}
or
add method getJobsRowComponents